tests/data_convert/test_tab_cleaner.py

```diff
@@ -102,7 +102,7 @@ def test_convert_types():
     assert(type(df.a[0]) == str)
     assert(df.a.tolist() == ["x", "y", "z"])
 
-    assert(type(df.b[0]) == np.int64)
+    assert(type(df.b[0]) == np.int64 or type(df.b[0]) == np.int32)
```

Comment:
what's this change doing?

Reply:
making the test work with both the 32bit and 64bit version of Pandas

Reply:
making the code work with both 32 and 64 bits pandas